README.md

```diff
@@ -73,9 +73,10 @@ To help inform our roadmap, keep adopters apprised of major updates and changes
 
 1. Ensure that the current version of Distributor is active on BOTH sites being connected.  We'll refer to these as mainsite.com and remotesite.com.
 1. On mainsite.com, navigate to `Distributor` > `External Connections` and click `Add New`.
-1. Enter a label for the connection (e.g., `remotesite.com`), select `Username / Password` for the `Authentication Method`, and a username from remotesite.com.
-1. On remotesite.com, ensure that [Application Passwords](https://wordpress.org/plugins/application-passwords/) is installed. (_Note: Using this plugin instead of a normal WordPress users password helps limit the use of your primary password and will allow you to revoke access to Distributor in the future if needed._) Then navigate to the user profile that will be used to create the External Connection on mainsite.com and then to the `Application Passwords` section of the user profile (not the `Account Management` section).  Add a label for the New Application Password Name (e.g., `mainsite.com`) and click `Add New`.  Now copy the password provided into mainsite.com's External Connections `Password` field.
-1. On mainsite.com, add the `External Connection URL` (e.g., http://remotesite.com/wp-json).  You should see a green circle and "_Connection established._".
+1. Enter a label for the connection (e.g., `remotesite.com`).
+1. Select `Username / Password` for the `Authentication Method`, open the `manually set up connection` link, and input a username from remotesite.com.
```

Comment:
This and the next couple steps can generally be skipped as the "Remote Connection Wizard" helps skip many of these steps.  Should essentially be (1) add label for the External Connection, (2) enter domain for the external site, (3) click Authorize Connection, (4) process acceptance from there to get to the final step of saving the new External Connection.

Reply:
@skorasaurus good callout on the text on the Edit External Connection page, we could/should probably remove that entire line at this point as that adds unnecessary plugins to someone's site as WordPress core now handles all that functionality.  If you'd like, I'd say go ahead and strip that out as part of this PR as well if you'd like?